docs/cristin.ttl

```diff
@@ -906,10 +907,81 @@ It is a self reference.""" ;
         entityDescription.reference.doi
     """ .
 
+:antall_sider_totalt
+    a               :cristinfield ;
+    rdfs:comment    "The total number of pages. Possibly null." ;
+    rdfs:range      rdf:nil ; #unknown range
+    rdfs:domain     :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :isignored   "true" .
+
+:publiseringsar
+    a               :cristinfield ;
+    rdfs:comment    "The year of publication for the journal" ;
+    rdfs:range      xsd:positiveInteger ;
+    rdfs:domain     :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :isignored   "true" .
+
+:hefte
+    a               :cristinfield ;
+    rdfs:comment    "The booklet containing the publication. Data in the form of a number stored as a string" ;
+    rdfs:range      xsd:string ;
+    rdfs:domain     :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :isignored   "true" .
+
+:status_referee_ordning
+    a               :cristinfield ;
+    rdfs:comment    "Defines whether or not the Journal is peerreviewed (J)=yes, (N)=no and NULL." ;
+    rdfs:range      xsd:string ;
+    rdfs:domain     :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :isignored   "true" .
+
+:supplement
+    a               :cristinfield ;
+    rdfs:comment    "A area for adding information about possible attachments to the article." ;
+    rdfs:range      xsd:string ;
+    rdfs:domain     :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :isignored   "true" .
+
+:utbredelsesomrade
+    a               :cristinfield ;
+    rdfs:comment    """
+        Information about where or to what extent the article has been published.
+        Examples: NORSK, NORDISK, INTERNASJ, NASJONALT, UKJENT""" ;
+    rdfs:range      xsd:string ;
+    rdfs:domain     :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :isignored   "true" .
+
+:artikkelnummer
+    a               :cristinfield ;
+    rdfs:comment    """
+        A number identifying the article within the journal.
+        This might be a sequence number or the actuall ID of the article, this is decided by the journal
+        The ID consists of a combination of letters and numbers or just numbers.
+        Examples: L08802, 1588, 2.
+        Possibly null.
+        """ ;
+    rdfs:range      xsd:string ;
+    rdfs:domain     :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :isignored   "true" .
+
+:arstall_trykket
+    a               :cristinfield ;
+    rdfs:comment    "The year the publication was printed." ;
+    rdfs:range      xsd:positiveInteger ;
+    rdfs:domain     :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :isignored   "true" .
+
+:arstall_online
+    a               :cristinfield ;
+    rdfs:comment    "The year the publication was released online." ;
+    rdfs:range      xsd:positiveInteger ;
+    rdfs:domain     :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :isignored   "true" .
+
 :TIDSSKRIFT
     a               :cristinfield ;
     rdfs:comment    """
-    UNKNOWN.
+    A journal, in this case the journal in which the publication was published.
```

Comment:
Better to move this description in the class definition and here just refer to the range class. "e.g. see range class description"